modules/customize-certificates-add-service-serving-crd.adoc

```diff
@@ -3,9 +3,9 @@
 // * security/certificates/service-serving-certificate.adoc
 
 [id="add-service-certificate-crd_{context}"]
-= Add the service CA bundle to a Custom Resource Definition
+= Add the service CA bundle to a custom resource definition
 
-You can annotate a Custom Resource Definition (CRD) with `service.beta.openshift.io/inject-cabundle=true` to have its `spec.conversion.webhook.clientConfig.caBundle` field populated with the service CA bundle. This allows the Kubernetes API server to validate the service CA certificate used to secure the targeted endpoint.
+You can annotate a `CustomResourceDefinition` (CRD) object with `service.beta.openshift.io/inject-cabundle=true` to have its `spec.conversion.webhook.clientConfig.caBundle` field populated with the service CA bundle. This allows the Kubernetes API server to validate the service CA certificate used to secure the targeted endpoint.
```

Comment:
In the OLM docs, @adellape is using _custom resource definition (CRD)_  I believe. Are we applying (CRD) to the object version as well?

Reply:
Chatted with @adellape who said that this seemed fine to him, so I'm going to leave it.